shared/teams/team/invites/invite-history.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as Constants from '../../../constants/teams'
+import * as Types from '../../../constants/types/teams'
+import * as TeamsGen from '../../../actions/teams-gen'
+import {memoize} from '../../../util/memoize'
+import {Section} from '../../../common-adapters/section-list'
+import {formatDurationLong} from '../../../util/timestamp'
+import {useTeamDetailsSubscribe} from '../../subscriber'
+import {ModalTitle} from '../../common'
+
+type Props = Container.RouteProps<{teamID: Types.TeamID}>
+
+const splitInviteLinks = memoize((inviteLinks: Set<Types.InviteLink>): {
+  active: Array<Types.InviteLink>
+  expired: Array<Types.InviteLink>
+} => ({active: [...inviteLinks].filter(i => !i.expired), expired: [...inviteLinks].filter(i => i.expired)}))
+
+const InviteHistory = (props: Props) => {
+  const teamID = Container.getRouteProps(props, 'teamID', Types.noTeamID)
+  useTeamDetailsSubscribe(teamID)
+  const teamDetails = Container.useSelector(s => Constants.getTeamDetails(s, teamID))
+  const loading = teamDetails === Constants.emptyTeamDetails // TODO should be a better way to check this
+  const [showingExpired, setShowingExpired] = React.useState(false)
+
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const onClose = () => dispatch(nav.safeNavigateUpPayload())
+  const onGenerate = () => {} // TODO
+
+  const {inviteLinks} = teamDetails
+  const {active, expired} = splitInviteLinks(inviteLinks)
+  const data: Array<Types.InviteLink> = showingExpired ? expired : active
+  const sections: Array<Section<Types.InviteLink>> = [
+    {
+      data,
+      key: 'invites',
+    },
+  ]
+
+  const emptyOrLoading =
+    loading || !data.length ? (
+      <Kb.Box2
+        direction="vertical"
+        fullWidth={true}
+        style={Styles.globalStyles.flexOne}
+        centerChildren={true}
+      >
+        {loading ? (
+          <Kb.Box2 direction="vertical" fullWidth={true} gap="tiny" centerChildren={true}>
+            <Kb.ProgressIndicator type="Large" />
+            <Kb.Text type="BodySmall">Loading...</Kb.Text>
+          </Kb.Box2>
+        ) : (
+          <Kb.Box2 direction="vertical" fullWidth={true} gap="tiny" centerChildren={true}>
```

Comment:
This empty state doesn't appear for me.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/4698687/78720796-50224d00-78f4-11ea-82e6-452d80e1515a.png">

Reply:
I can't repro this, are you possibly out of date?

Reply:
the most recent commit adds the empty state